### PR TITLE
React with webpacker

### PR DESCRIPTION
**Pull request by marcomo**
This adds `webpacker`, `react-rails`, and `foreman`.
- `webpacker` compiles/bundles JS modules.
- `react-rails` provides react component integration with Rails.
- `foreman` handles running rails and webpack servers.

This update also converts the rails views for issues#index into react components.

### CODE REVIEW

File: app/controllers/issues_controller.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class IssuesController < ApplicationController
 
   def index
     @issues = {}
+    puts "current user: #{current_user.id}"
```
Comment (crustyratfink) [inline app/controllers/issues_controller.rb]: Is this for testing? Could we do a logger.debug?

File: app/models/consumer_user.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 class ConsumerUser < User
-  belongs_to :consumer, class_name: "Consumer", foreign_key: "company_id"
+  belongs_to :consumer, class_name: "Consumer", foreign_key: "user_id"
```
Comment (crustyratfink) [inline app/models/consumer_user.rb]: I'm going to rename consumer to consumercompany, but it's a company

Comment (marcomo) [inline app/models/consumer_user.rb]: 👍

File: db/seeds.rb [hunk 3]
```diff
@@ -89,12 +89,12 @@
   Issue.create!(
             title: "Test Issue ##{id}",
             body: "I would like you to do something.",
-            requester_id: requester.id,
+            requester_id: 2,
```
Comment (crustyratfink) [inline db/seeds.rb]: This is brittle... cuz we don't know what these IDs will be. Could we just:
  requester = consumer_users[1]
  assignee = provider_users[1]

Comment (crustyratfink) [inline db/seeds.rb]: I'll pull it and fix it.

Comment (marcomo) [inline db/seeds.rb]: Oops. My bad. Put that in there to force them to specific logins. I can patch that.

File: app/controllers/issues_controller.rb (hunk 1 above)
Comment (marcomo) [inline app/controllers/issues_controller.rb]: It can be removed.

**Pull request merged**